src/main/java/com/laytonsmith/core/events/drivers/InventoryEvents.java

```diff
@@ -938,6 +943,7 @@ public Map<String, Mixed> evaluate(BindableEvent event) throws EventException {
 				for(int i = 0; i < mi.length; i++) {
 					matrix.set(i, ObjectGenerator.GetGenerator().item(mi[i], t), t);
 				}
+				ret.put("inventorytype", new CString(e.getInventory().getType().toString(), t));
```

Comment:
This isn't documented. However, is this even needed for something? It only returns CRAFTING or WORKBENCH right now, which can already be inferred by the matrix size, which is the only meaningful difference as far as this event is concerned. Maybe if this fired for CRAFTER too, but it doesn't. 

In any case, one of the other needs to change: document this or remove it.

Reply:
Removed.